poetry.lock

```diff
@@ -1,9 +1,10 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.4.0 and should not be changed by hand.
```

Comment:
Please update your poetry version and run `poetry lock`

Reply:
Sorry, but according to RecTools [contribute guide](https://github.com/MobileTeleSystems/RecTools?tab=readme-ov-file#contribution): 
> you must have python3 and poetry==1.4.0 installed

So, that's why I don't update my poetry version. Of course I can, but let's clarify this moment before.

Reply:
It's a good point, thanks

Could you please fix it in the contribute guide as well, say `python>=3.8 and poetry>=1.5.0`

Reply:
In both: contribution guide itself and readme

Reply:
Done. Update both files and lock poetry.

After the update the linter tests for `tests/dataset/test_torch_dataset.py` failed, I will look into what happened. I guess the update of torch version when executing `poetry lock` has affected it so much

Reply:
Fixed